release/downloadIstioCtl.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Separate downloader for istioctl
+#
+# This file will be fetched as: curl -L https://git.io/getLatestIstioCtl | sh -
+#
+
+# Determines the operating system.
+OS="$(uname)"
+if [ "x${OS}" = "xDarwin" ] ; then
+  OSEXT="osx"
+else
+  OSEXT="linux"
+fi
+
+# Determines the istioctl version.
+if [ "x${ISTIO_VERSION}" = "x" ] ; then
+  ISTIO_VERSION=$(curl -L -s https://api.github.com/repos/istio/istio/releases | \
+                  grep tag_name | sed "s/ *\"tag_name\": *\"\\(.*\\)\",*/\\1/" | \
+                  sort -t"." -k 1,1 -k 2,2 -k 3,3 -k 4,4 | tail -n 1)
+fi
+
+if [ "x${ISTIO_VERSION}" = "x" ] ; then
+  printf "Unable to get latest Istio version. Set ISTIO_VERSION env var and re-run. For example: export ISTIO_VERSION=1.0.4"
+  exit;
+fi
+
+# Downloads the istioctl binary archive.
+tmp=$(mktemp -d /tmp/istioctl.XXXXXX)
+filename="istioctl-${ISTIO_VERSION}-${OSEXT}.tar.gz"
+cd "$tmp"
+URL="https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istioctl-${ISTIO_VERSION}-${OSEXT}.tar.gz"
+printf "Downloading %s from %s ..." "filename" "$URL"
+curl -LO "${URL}"
+printf "istioctl %s Download Complete!, Validating Checksum...\n" "$ISTIO_VERSION"
+
+
+# Validate Checksum
+SHA=$(curl -sL "${URL}.sha256" | cut -d ' ' -f1)
+checksum=$(openssl dgst -sha256 "${filename}" | cut -d' ' -f2)
```

Comment:
is openssl standard?

Reply:
according to wikipedia:
`OpenSSL is available for most Unix-like operating systems (including Linux, macOS, and BSD) and Microsoft Windows.`

Reply:
We can actually skip the checksum validation as well and let user do this by themself using the sha we published.

Reply:
```
$ docker run -it ubuntu:bionic
root@0023bb6006e3:/# openssl
bash: openssl: command not found
```

tells me it is not standard.

`sha256sum` does seem to be, not sure if it is on mac though

Reply:
It does not have sha256sum on mac. Based on that, Let's remove the checksum validation part.

Reply:
removed the checksum validation function